src/widget/wtrackmenu.h

```diff
@@ -67,6 +67,7 @@ class WTrackMenu : public QMenu {
     // WARNING: This function hides non-virtual QMenu::popup().
     // This has been done on purpose to ensure menu doesn't popup without loaded track(s).
     void popup(const QPoint& pos, QAction* at = nullptr);
+    void slotShowTrackInfo();
```

Comment:
Are we okay with exposing (changing private -> public) the internal slot of `WTrackMenu`? I understand that displaying `DlgTrackInfo` is a common requirement for various widgets, but should it be accessed via `WTrackMenu`?

Reply:
what's the downside?
the reason for taking this shortcut was that when trying to open `DlgTrackInfo` directly I got compiler errors when I tried to pass TrackPointer to `DlgTrackInfo::loadTrack` because [there are two functions with the same name](https://github.com/mixxxdj/mixxx/blob/master/src/library/dlgtrackinfo.h#L37):
`void loadTrack(TrackPointer pTrack)` <- what we want
`void loadTrack(QModelIndex index)` <- picked automatically

`WTrackMenu` otoh has different function for TrackPointer and trackIndexList: https://github.com/mixxxdj/mixxx/blob/master/src/widget/wtrackmenu.h#L57

Is the shortcut okay or should I refactor `DlgTrackInfo` like `WTrackMenu`?

Reply:
I think we should use the shortcut and re-use the instance already available in WTrackMenu instead of creating a new one, right?

Reply:
Okay. That would be easier and won't require much re-implementation since everything is already set up within the context of the menu.